Source/TAS/Manager.cs

```diff
@@ -27,6 +27,14 @@ public enum State
 
     internal static Save.LevelRecord TASLevelRecord = new();
 
+    public static Vec3 FreeCamPosition;
+    public static Vec2 FreeCamRotation;
+    public static float FreeCamDistance = 50f;
+
+    public static Vec2 nextMousePosition;
+    public static Vec2 prevMousePosition;
+    public static Vec2 MouseDelta => Foster.Framework.Input.Mouse.Position - prevMousePosition;
```

Comment:
I realized we can just do `Input.Mouse.Position - Input.LastState.Mouse.Position` for the delta, so next/prevMousePosition could be removed.